llvm/lib/Transforms/IPO/RepoPruning.cpp

```diff
@@ -295,6 +296,19 @@ static bool eliminateUnreferencedAndPrunedGOs(Module &M) {
   return Changed;
 }
 
+static llvm::Value *createDeclararion(Module &M, llvm::Type *Ty,
+                                      const Module::alias_iterator &It) {
+  llvm::Value *Declaration;
+  if (FunctionType *FTy = dyn_cast<FunctionType>(Ty)) {
+    Declaration = Function::Create(FTy, GlobalValue::ExternalLinkage,
```

Comment:
Could we have an early return here to avoid having the uninitialized variable "Declaration" above?

Reply:
Thanks Paul. The issue has been fixed.